contributed_definitions/NXmpes.nxdl.xml

```diff
@@ -377,6 +406,37 @@ Optional constant settings (like lens focusing parameters on the sample: positio
                     </doc>
                 </field>
             </group>
+            <group name="energy_correction" type="NXcalibration" optional="true">
+                <field name="applied" type="NX_BOOLEAN">
+                    <doc>
+                         Have the energy axes been adjusted (e.g., in cases where samples are not
+                         conductive)?
+                    </doc>
+                </field>
+                <field name="emission_line" recommended="true">
+                    <doc>
+                         Emission line which was used for the calibration
+                    </doc>
+                </field>
```

Comment:
It would be good to have a concept of an XPS region (or X-ray edge) because this can also be stored on the NXmpes level. See also [here](https://github.com/nexusformat/definitions/pull/1293).

For now, I have just put some examples.

Reply:
Or rather `NXcore_level` instead of `NXxray_edge`

Reply:
See also https://github.com/nexusformat/definitions/pull/1293